nncf/common/accuracy_aware_training/runner.py

```diff
@@ -127,8 +127,8 @@ def initialize_training_loop_fns(
         validate_fn: Callable[[TModel, Optional[int]], float],
         configure_optimizers_fn: Callable[[], Tuple[OptimizerType, LRSchedulerType]],
         dump_checkpoint_fn: Callable[[TModel, CompressionAlgorithmController, "TrainingRunner", str], None],
-        **kwargs,
-    ):
+        **kwargs: Any,
```

Comment:
This type hint is incorrect, review your general Python knowledge as to what type the `**kwargs` parameter is constrained to.